setup.py

```diff
@@ -64,26 +63,22 @@ def read(*rnames):
         return f.read()
 
 
-codeoptimization = Feature(
-    "Optional code optimizations",
-    standard=True,
-    ext_modules=[
+codeoptimization = [
         Extension(
             "zope.hookable._zope_hookable",
             [os.path.join('src', 'zope', 'hookable', "_zope_hookable.c")],
         ),
-    ],
-)
+]
 
 is_pypy_or_jython = platform.python_implementation() in ('PyPy', 'Jython')
 
 # Jython cannot build the C optimizations, while on PyPy they are
 # anti-optimizations (the C extension compatibility layer is known-slow,
 # and defeats JIT opportunities).
 if is_pypy_or_jython:
```

Comment:
Maybe please add an or is_pure check with the os.getenv('PURE_PYTHON') logic copied from zope.interface etc.? So people can ask to skip extension modules.

Reply:
@mgedmin Based on what we decided in zope.interface and zope.i18nmessageid, is there anything that still needs done here?

Reply:
A changelog entry would be nice!